projects/common/02drv_sensors/02drv_sensors.c

```diff
@@ -0,0 +1,64 @@
+/**

```

Comment:
@peretuset,
@NicolaAccettura pointed out that this sample applications only runs on the OpenMoteCC2538. Would it be possible to move into the board-specific project directory.